src/renderer/store/editor.js

```diff
@@ -39,6 +40,14 @@ const state = {
   platform: process.platform
 }
 
+const getMarkdownWithEncoding = state => {
```

Comment:
I think put `getMarkdownWithEncoding ` in `main` folder is better,

So `add/remove` `0xFEFF` are at the same place.

Reply:
Yep, I also thought about it and pass `isUtf8Bom` parameter also.